curvefs/proto/metaserver.proto

```diff
@@ -54,12 +54,18 @@ message GetDentryRequest {
     optional uint64 appliedIndex = 8;
 }
 
+enum DentryFlag {
+    TYPE_FILE_FLAG = 1;
+    DELETE_MARK_FLAG = 2;
+}
+
 message Dentry {
     required uint32 fsId = 1;
     required uint64 inodeId = 2;
     required uint64 parentInodeId = 3;
     required string name = 4;
     required uint64 txId = 5;
+    optional uint32 flag = 6;
```

Comment:
use enum type ?

Reply:
> use enum type ?

The `flag` is a set of `DentryFlag` which is enum, like:
```
 flag = TYPE_FILE_FLAG | DELETE_MARK_FLAG
```